Gemfile.lock

```diff
@@ -437,4 +440,4 @@ DEPENDENCIES
   webmock
 
 BUNDLED WITH
-   1.17.3
+   2.0.2
```

Comment:
any consequences we should be worried about with the major version bump?

Reply:
Tests all pass and it's probably a good idea anyway. 🤷‍♂

Reply:
scratch that, these lock file was a result of a reverted bundler update.